src/libs/actions/User.js

```diff
@@ -174,6 +179,25 @@ function validateLogin(accountID, validateCode) {
     });
 }
 
+/**
+ * Checks if the expiresAt date of a user's ban is before right now
+ *
+ * @param {String} expiresAt
+ * @returns {boolean}
+ */
+function isBlockedFromConcierge(expiresAt) {
+    if (!expiresAt) {
+        return false;
+    }
+
+    const now = moment()
+        .format('YYYY-MM-DD');
+    const expires = moment(expiresAt)
+        .format('YYYY-MM-DD');
+
+    return now < expires;
```

Comment:
I feel like this can be simplified with moment's [comparison methods](https://momentjs.com/docs/#/query/), i.e. the whole method can be rewritten to something like

```
!_.isNull(expiresAt) && moment().isBefore(moment(expiresAt));
```
or to only use the date and not the time,
```
... moment().isBefore(moment(expiresAt), 'day');
```